packages-next/testing/src/admin-ui-test-utils.ts

```diff
@@ -0,0 +1,115 @@
+import path from 'path';
```

Comment:
This shouldn't live in the testing package, I don't think we want to make this a public API right now

Reply:
I'm fine to remove this for now, but curious as to what's stopping us from making this public API?

Reply:
Nothing is _stopping us_ but a public API means that we need to support it, document it, etc. and given consumers shouldn't need to use this API, let's not make it public